tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -81,14 +108,17 @@ def __init__(self, vm: VmIdentifier, paths: AgentTestContext.Paths, connection:
             self.log: Logger = None
             self.node: Node = None
             self.runbook_name: str = None
-            self.suite_name: str = None
+            self.image_name: str = None
+            self.test_suites: List[str] = None
+            self.collect_logs: str = None
+            self.skip_setup: bool = None
```

Comment:
Sorry, don't get your question... Previously the runbook was executed all the tests (methods marked with TestMetadata), now it executes the tests passed as parameters in the 'test_suites' variable

Reply:
My question is this change the behavior of single test run that we execute from command line (developer scenario) like running vmaccess.py etc.

Reply:
I see. Without my changes, one cannot pass the test to run as parameter. The runbook executes all the methods marked as tests with metadata. With my changes, one can pass the test suite to run and the runbook will execute only that suite. To execute a single test instead of a test suite, one can pass a piece of JSON describing a test suite comprised of that single test, e.g. { "name": "MyTestRun", "tests": [ "bvts/vmaccess.py"] } (however, the current pipeline has a bug and it cannot take spaces in is parameters, that is already fixed in my next PR. the parameter for the runbook itself can take spaces)